### PR TITLE
Update for Tesseract-ocr 3.04.00 release

### PR DESCRIPTION
**Pull request by Shreeshrii**
More details at

https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/tesseract-dev/8e0F2cK2YzU/EHH59yMJIowJ

### CODE REVIEW

File: mingw-w64-tesseract-ocr/PKGBUILD [hunk 1]
```diff
@@ -17,11 +17,11 @@ depends=(${MINGW_PACKAGE_PREFIX}-gcc-libs
         ${MINGW_PACKAGE_PREFIX}-pango
         ${MINGW_PACKAGE_PREFIX}-zlib)
 options=('!libtool' 'strip')
-source=("https://github.com/tesseract-ocr/tesseract/archive/master.zip")
+source=("https://github.com/tesseract-ocr/tesseract/archive/${pkgver}.zip")
 sha256sums=('SKIP'
 )
 prepare() {
-  cd "$srcdir/${_realname}-master"
+  cd "$srcdir/${_realname}-/${pkgver}"
```
Comment (Alexpux) [inline mingw-w64-tesseract-ocr/PKGBUILD]: Why need "/" before ````pkgver````?

File: mingw-w64-tesseract-ocr/PKGBUILD [hunk 2]
```diff
@@ -34,7 +34,7 @@ build() {
     extra_config+=( --enable-debug )
   fi
 
-  ${srcdir}/${_realname}-master/configure \
+  ${srcdir}/${_realname}-/${pkgver}/configure \
```
Comment (Alexpux) [inline mingw-w64-tesseract-ocr/PKGBUILD]: Why need "/" before ````pkgver````?

Comment (Shreeshrii) [inline mingw-w64-tesseract-ocr/PKGBUILD]: typo. please remove.

Comment (Shreeshrii) [inline mingw-w64-tesseract-ocr/PKGBUILD]: https://packages.debian.org/sid/tesseract-ocr

FYI - dependencies

**Pull request merged**